common.sh

```diff
@@ -144,6 +148,17 @@ function build_kb {
   GO111MODULE=on go build $opts -o $tmp_root/kubebuilder/bin/kubebuilder ./cmd
 }
 
+function fetch_kind {
+  header_text "Checking for kind"
```

Comment:
I think it is not required since we can call `./sccripts/install_and_setup.sh` to this setup. WDYT?

Reply:
I think, maybe I need to do some more work than I thought. I was duplicating it because in prow you only get one command, so unlike travis there is no "pre-run" of installing and setting up kind. 

I think I should probably make the whole invocation idempotent, handle the setup in a root script which invokes v1/v2 e2e test. Do you think that makes sense?

Reply:
I am working to try refac the scripts. I just started. See https://github.com/kubernetes-sigs/kubebuilder/pull/1151. ( note that it has the changes performed in the https://github.com/kubernetes-sigs/kubebuilder/pull/1135 as well for I am able to move forward and work with using go 1.13. ). Let's see if in this PR I will be able to reduce the complexity and remove the duplications which may help us better understand what/where is required.

Reply:
@mengqiy ^